browser/sources.gni

```diff
@@ -252,6 +252,32 @@ brave_chrome_browser_deps = [
   "//url",
 ]
 
+if (!is_android) {
+  # Already included on android by upstream
+
+  # Including v2 back into brave build as we make use of `feed::FetchRssLinks` in `brave_news_tab_helper.cc`.
+  brave_chrome_browser_sources += [
```

Comment:
I'm a bit worried about this longer term, should we be using something else for brave news @petemill ?